ArduCopter/mode_acro.cpp

```diff
@@ -57,6 +56,12 @@ void ModeAcro::run()
         // do nothing
         break;
     }
+}
+
+void ModeAcro::run()
+{
+    float target_roll_rads, target_pitch_rads, target_yaw_rads, pilot_desired_throttle;
+    set_desired_rates(target_roll_rads, target_pitch_rads, target_yaw_rads, pilot_desired_throttle);
```

Comment:
What am I not understanding here?  `set` with a bunch of uninitialised stack variables?